Update/omot_027.txt

```diff
@@ -831,20 +831,20 @@ void main()
 //rそして俺たちは、めいめい持ち合った弁当を広げて、わいわいと楽しく箸をつついては舌鼓を打ち合う。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "そして俺たちは、めいめい持ち合った弁当を広げて、わいわいと楽しく箸をつついては舌鼓を打ち合う。",
-			NULL, "Then, we spread our lunches out in front of us, and happily pecked at them with our chopsticks.", Line_Normal);
+			NULL, "Then, we spread our lunches out in front of us, and happily shared them and feasted with our chopsticks.", Line_Normal);
 	ClearMessage();
 
 //rやはり一番人気はレナスペシャルマーク　（本人命名）で、前々から目をつけていたのか下級生たちはそれに殺到し、その可愛らしさにレナ必殺の『お持ち帰りぃ～♪』が天下御免にまかり通っていた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-3>", Line_Continue); }
 	OutputLine(NULL, "やはり一番人気はレナスペシャルマーク　（本人命名）で、前々から目をつけていたのか下級生たちはそれに殺到し、その可愛らしさにレナ必殺の『お持ち帰りぃ～♪』が天下御免にまかり通っていた。",
-			NULL, "Of course, the most popular item was the (so called) Rena Trademark Special and the younger kids must have been eyeing it up, because they all immediately jumped on it. The cuteness of Rena's deadly \"I wanna take it home with mee~♪\" was smothering the entire world.", Line_Normal);
+			NULL, "Of course, the most popular meal was the (so called) Rena Trademark Special and the younger kids must have been eyeing it up, because they all immediately jumped on it. The cuteness of the kids made Rena's deadly \"I wanna take it home with mee~♪\" and it was smothering the entire world.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Of course, the most popular meal was the (so called) Rena Trademark Special and the younger kids must have been eyeing it up, because they all immediately jumped on it. The cuteness of the kids invoked Rena's deadly \"I wanna take you home with mee~♪\" and it was smothering the entire world.", Line_Normal);
```